<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbsUnitsFrame</class>
 <widget class="QFrame" name="AbsUnitsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="absunits_gb">
     <property name="title">
      <string>Perform Absolute Normalisation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="run_files_gb">
        <property name="title">
         <string>Run Files</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="absunits_van_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>AbsUnits Vanadium</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="absunits_van_edit"/>
           </item>
           <item>
            <widget class="QPushButton" name="absunits_van_browse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="grouping_file_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Grouping File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="grouping_file_edit"/>
           </item>
           <item>
            <widget class="QPushButton" name="grouping_file_browse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="absunits_detvan_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Detector Vanadium (Abs Units)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="absunits_detvan_edit"/>
           </item>
           <item>
            <widget class="QPushButton" name="absunits_detvan_browse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="integration_gb">
        <property name="title">
         <string>Integration (meV)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="ei_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Incident Energy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ei_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="emin_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Energy range                             E Min</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emin_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="emax_label">
             <property name="text">
              <string>E Max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emax_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="masses_gb">
        <property name="title">
         <string>Masses for Absolute Units</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="van_mass_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Vanadium Mass</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="van_mass_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="sample_mass_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Sample Mass</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sample_mass_edit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="sample_rmm_label">
             <property name="minimumSize">
              <size>
               <width>225</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Sample RMM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="sample_rmm_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="diag_gb">
        <property name="title">
         <string>Diagnostic Parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="median_test_out_high_label">
             <property name="minimumSize">
              <size>
               <width>137</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Median high outliers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="median_test_out_high_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="median_test_out_low_label">
             <property name="minimumSize">
              <size>
               <width>132</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Median low outliers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="median_test_out_low_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="median_test_high_label">
             <property name="minimumSize">
              <size>
               <width>137</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Median test high</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="median_test_high_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="median_test_low_label">
             <property name="minimumSize">
              <size>
               <width>132</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Median test low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="median_test_low_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="errorbar_crit_label">
             <property name="minimumSize">
              <size>
               <width>137</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Errorbar criterion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="errorbar_crit_edit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>962</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
